packages.md

```diff
@@ -179,10 +186,12 @@
+| [`Microsoft.Extensions.Azure`]( sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj )| [Readme](sdk/core/Microsoft.Extensions.Azure/README.md) | [Changelog](sdk/core/Microsoft.Extensions.Azure/CHANGELOG.md) | [![Microsoft.Extensions.Azure](https://img.shields.io/nuget/vpre/Microsoft.Extensions.Azure.svg)]( https://www.nuget.org/packages/Microsoft.Extensions.Azure/ ) |
 | [`Microsoft.Rest.ClientRuntime`]( sdk/mgmtcommon/ClientRuntime/ClientRuntime/Microsoft.Rest.ClientRuntime.csproj )|  N/A  |  N/A  | [![Microsoft.Rest.ClientRuntime](https://img.shields.io/nuget/vpre/Microsoft.Rest.ClientRuntime.svg)]( https://www.nuget.org/packages/Microsoft.Rest.ClientRuntime/ ) |
 | [`Microsoft.Rest.ClientRuntime.Azure`]( sdk/mgmtcommon/ClientRuntime.Azure/ClientRuntime.Azure/Microsoft.Rest.ClientRuntime.Azure.csproj )|  N/A  |  N/A  | [![Microsoft.Rest.ClientRuntime.Azure](https://img.shields.io/nuget/vpre/Microsoft.Rest.ClientRuntime.Azure.svg)]( https://www.nuget.org/packages/Microsoft.Rest.ClientRuntime.Azure/ ) |
 | [`Microsoft.Rest.ClientRuntime.Azure.Authentication`]( sdk/mgmtcommon/Auth/Az.Auth/Az.Authentication/Microsoft.Rest.ClientRuntime.Azure.Authentication.csproj )|  N/A  |  N/A  | [![Microsoft.Rest.ClientRuntime.Azure.Authentication](https://img.shields.io/nuget/vpre/Microsoft.Rest.ClientRuntime.Azure.Authentication.svg)]( https://www.nuget.org/packages/Microsoft.Rest.ClientRuntime.Azure.Authentication/ ) |
 | [`Microsoft.Rest.ClientRuntime.Azure.TestFramework`]( sdk/mgmtcommon/TestFramework/ClientRuntime.Azure.TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework.csproj )|  N/A  |  N/A  | [![Microsoft.Rest.ClientRuntime.Azure.TestFramework](https://img.shields.io/nuget/vpre/Microsoft.Rest.ClientRuntime.Azure.TestFramework.svg)]( https://www.nuget.org/packages/Microsoft.Rest.ClientRuntime.Azure.TestFramework/ ) |
 | [`Microsoft.Rest.ClientRuntime.Etw`]( sdk/mgmtcommon/ClientRuntime.Etw/Microsoft.Rest.ClientRuntime.Etw.csproj )| [Readme](sdk/mgmtcommon/ClientRuntime.Etw/README.md) |  N/A  | [![Microsoft.Rest.ClientRuntime.Etw](https://img.shields.io/nuget/vpre/Microsoft.Rest.ClientRuntime.Etw.svg)]( https://www.nuget.org/packages/Microsoft.Rest.ClientRuntime.Etw/ ) |
 | [`Microsoft.Rest.ClientRuntime.Log4Net`]( sdk/mgmtcommon/ClientRuntime.Log4Net/Microsoft.Rest.ClientRuntime.Log4Net.csproj )| [Readme](sdk/mgmtcommon/ClientRuntime.Log4Net/README.md) |  N/A  | [![Microsoft.Rest.ClientRuntime.Log4Net](https://img.shields.io/nuget/vpre/Microsoft.Rest.ClientRuntime.Log4Net.svg)]( https://www.nuget.org/packages/Microsoft.Rest.ClientRuntime.Log4Net/ ) |
+| [`SnippetGenerator`]( eng/SnippetGenerator/SnippetGenerator.csproj )|  N/A  |  N/A  |  N/A  |
```

Comment:
These 2 should be excluded.

Reply:
Excluded `SnippetGenerator` and `docgen` from docsettings.